src/Docker/Runner/Authorization.php

```diff
@@ -45,26 +45,27 @@ public function getAuthorization($configData)
         $data = [];
         if (isset($configData['oauth_api']['credentials'])) {
             $data['oauth_api']['credentials'] = $configData['oauth_api']['credentials'];
-        }
-        if (isset($configData['oauth_api']['version']) && ($configData['oauth_api']['version'] == 3)) {
-            $client = $this->oauthClientV3;
         } else {
-            $client = $this->oauthClient;
-        }
-        if (isset($configData['oauth_api']['id'])) {
-            // read authorization from API
-            try {
-                $credentials = $client->getDetail(
-                    $this->componentId,
-                    $configData['oauth_api']['id']
-                );
-                $decrypted = $this->encryptor->decrypt($credentials);
-                $data['oauth_api']['credentials'] = $decrypted;
-            } catch (RequestException $e) {
-                if (($e->getCode() >= 400) && ($e->getCode() < 500)) {
-                    throw new UserException($e->getMessage(), $e);
-                } else {
-                    throw new ApplicationException($e->getMessage(), $e);
+            if (isset($configData['oauth_api']['version']) && ($configData['oauth_api']['version'] == 3)) {
```

Comment:
zvladl bys pridat nekam semka https://github.com/keboola/docker-bundle/blob/master/Tests/Runner/AuthorizationTest.php#L183 kontrolu ze se getDetail nezavola ?

Reply:
Myslim, ze ano.

Reply:
Je to tam